Sources/Brave/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2412,7 +2412,11 @@ public class BrowserViewController: UIViewController {
 extension BrowserViewController {
   func didScanQRCodeWithURL(_ url: URL) {
     popToBVC()
-    finishEditingAndSubmit(url, visitType: .typed)
+    let overlayText = URLFormatter.formatURL(url.absoluteString, formatTypes: [], unescapeOptions: [])
+
+    DispatchQueue.main.asyncAfter(deadline: .now() + 0.9) {
```

Comment:
Added a delay (as small as I found possible), otherwise url bar stays hidden behind the keyboard. Is there a better way to prevent this bug?

Reply:
Its a bug related to bottom bar movements while other screens are presented (https://github.com/brave/brave-ios/issues/7280, https://github.com/brave/brave-ios/issues/6517)

One way around this could be to add a completion block to the `popToBVC` so that we can execute when the controller is fully dismissed